pkg/kube/translation/apisix_route.go

```diff
@@ -977,13 +964,6 @@ func (t *translator) translateHTTPRouteV2beta3NotStrictly(ctx *TranslateContext,
 		route := apisixv1.NewDefaultRoute()
 		route.Name = apisixv1.ComposeRouteName(ar.Namespace, ar.Name, part.Name)
 		route.ID = id.GenID(route.Name)
-		pluginConfig := apisixv1.NewDefaultPluginConfig()
```

Comment:
Why we don't handle the `PluginConfigName` in this method?

Reply:
That's because when it creates the ApisixRoute and the field `PluginConfigName` has been specified, I want to use apisix.Cluster api to check the plugin was whether exist.
 And at the same time replace it with the plugin id.
...
Since the method to generate the plugin id is fixed.
I will just remove the replacement and remain checking.

Reply:
Well, I mean `translateHTTPRouteV2beta3NotStrictly` is basically same to `translateHTTPRouteV2beta3`, so here you should also set the plugin config id.